Content.Server/Fluids/EntitySystems/AbsorbentSystem.cs

```diff
@@ -117,30 +117,43 @@ private bool TryTransferAbsorber(EntityUid user, EntityUid used, EntityUid targe
         if (!_solutionSystem.TryGetRefillableSolution(target, out var refillableSolution, refillable: refillable))
             return false;
 
-        if (refillableSolution.Volume <= 0)
+        if (refillableSolution.Volume < 0)
         {
             var msg = Loc.GetString("mopping-system-target-container-empty", ("target", target));
             _popups.PopupEntity(msg, user, user);
             return false;
         }
 
+        var transferAmount = component.PickupAmount / PuddleSystem.EvaporationReagentRatio -
+            absorberSoln.GetReagentQuantity(PuddleSystem.EvaporationReagent);
```

Comment:
The transfer amount shouldn't be affected by any sort of reagent ratio as then you're effectively just changing the transfer amount.

Additionally the mop is now unavailable to fill up and only goes to 1/3.

Reply:
The split-bar needs adjusting to show the portion of water filled up as right now it looks bugged only going to n% instead of the full bar.

Reply:
That's exactly _what_ the issue is changing: The amount of puddle volume someone can transfer with a given amount of water. It is supposed to change transfer amount.

The mop only fills to 1/3 cause otherwise you wouldn't be able to fill it with puddle volume, since now you don't need 60u water to clean 3 20u volume puddles. If it were filled with 60u water you would be unable to clean properly, since nothing would remove all that amount of water from the mop, since mopping does not require just as much volume in water as the puddles it's cleaning.

It's not bugged, it's working exactly as intended. Now, it wasn't obvious to me either at the first time I play tested it. But if you try to mop some puddles you see it is working as intended: you "fill" the mop to 1/3 water, mop once it goes to 2/9, then 1/9, and after that the mop is full with puddle volume.

Numerically:
It starts with 20u water, 1/3 of the 60u capacity for the mop.
Since puddles can usually only hold 20u, then the first mopping only takes 20/3 water.
That would amount to 6.66, exactly, as truncated in FixedPoint2. So each mopping interaction transfers 6.66u water into the puddle and transfers 6.66 x 3 puddle volume to the mop. Now here you see numeric division problem arising: 6.66 x 3 equals 19.98, not twenty. This is normal, it's just how division works over scalar numbers in computers, it can't be exact. Now since I've made it transfer (6.66 + 0.01) water at each interaction by adding one epsilon, it will sum up to 20.01 instead. I did this for no other reason than to allow the mop to fill to 60u, instead of only filling up to 59.54. 

So the split-bar actually shows exactly what it is supposed to show.